packages/teams-js/src/private/videoEx.ts

```diff
@@ -80,9 +83,24 @@ export namespace videoEx {
     audioInferenceResult?: Uint8Array;
   }
 
+  /**
+   * Old video frame data structure, almost identical to the {@link VideoBufferData} except `videoFrameBuffer` is named as `data`.
+   * Old host like the old Teams passes this data to the SDK. It will be deprecated in the future.
+   */
+  type LegacyVideoBufferData = Omit<VideoBufferData, 'videoFrameBuffer'> & {
+    /**
+     * Video frame buffer
+     */
+    data: Uint8ClampedArray;
+  };
+
   /**
    * @hidden
-   * Video frame call back function
+   * Video frame call back function definition
+   * The callback will be called on every frame when running on the supported host.
+   * We require the frame rate of the video to be at least 22fps for 720p, thus the callback should process a frame timely.
```

Comment:
This line of documentation doesn't seem very helpful for developers calling the function, please consider removing it.

Reply:
Same for other copies of this line on other functions.

Reply:
removed